### PR TITLE
Less duplication in test code

### PR DESCRIPTION
**Pull request by gilescope**
Less duplication makes it clear what's important in each test.

### CODE REVIEW

File: client/transaction-pool/graph/src/base_pool.rs [hunk 1]
```diff
@@ -538,6 +538,19 @@ mod tests {
 		BasePool::default()
 	}
 
+	// (DEFAULT_TX can be static as the vec![] are empty and thus don't allocate.)
```
Comment (bkchr) [inline client/transaction-pool/graph/src/base_pool.rs:541]:
```suggestion
```

File: client/transaction-pool/graph/src/base_pool.rs [hunk 2]
```diff
@@ -538,6 +538,19 @@ mod tests {
...
+	static DEFAULT_TX : Transaction::<Hash, Vec<u8>> = Transaction {
```
Comment (bkchr) [inline client/transaction-pool/graph/src/base_pool.rs:542]:
```suggestion
	const DEFAULT_TX: Transaction::<Hash, Vec<u8>> = Transaction {
```

Comment (gilescope) [reply client/transaction-pool/graph/src/base_pool.rs:542]: Even nicer!

**Review by seunlanlege — APPROVED** [review]
yay for fewer lines of code 🎊

**Review by bkchr — APPROVED** [review]

**Review by expenses — APPROVED** [review]
👍🏻 👍🏻

**Pull request merged**